.github/CONTRIBUTING.rst

```diff
@@ -6,7 +6,7 @@ Build Status
 
 - master
 
-  + `Stable buildbots <http://buildbot.python.org/3.x.stable/>`_
+  + `Stable buildbots <http://buildbot.python.org/all/#/masters/>`_
 
 - 3.6
```

Comment:
You also need to fix the links of 3.6 and 2.7 branches.

Reply:
@berkerpeksag 👍 Yes. I didn't find links that matches.